<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkTbssSkeletonizationViewControls</class>
 <widget class="QWidget" name="QmitkTbssSkeletonizationViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>811</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <item>
    <widget class="QPushButton" name="m_Skeletonize">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Skeletonize</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_Project">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Skeletonize and Project</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_OutputMask">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Output binary mask</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_OutputDistanceMap">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Output distance map</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
